tests/test_cnn_cfg.py

```diff
@@ -1,19 +1,36 @@
 import torch
 import torch.nn as nn
 
+from rl_algorithms.common.networks.backbones.cnn import CNN
+from rl_algorithms.common.networks.backbones.resnet import ResNet
```

Comment:
same comment: https://github.com/medipixel/rl_algorithms/pull/209/files#r411047082

Reply:
resolved in commit 2cad1a7!!